types/v56/nsxt_types.go

```diff
@@ -1570,3 +1570,36 @@ type VdcNetworkProfileServicesEdgeCluster struct {
 	BackingID      string            `json:"backingId"`
 	EdgeClusterRef *OpenApiReference `json:"edgeClusterRef,omitempty"`
 }
+
+// NsxtEdgeGatewayQosProfiles defines a Gateway QoS Profile Object (structure comes from NSX-T)
+// This is a read-only entity in VCD
+type NsxtEdgeGatewayQosProfile struct {
+	// ID of the gateway QoS profile.
+	ID string `json:"id"`
+
+	DisplayName string `json:"displayName"`
+	Description string `json:"description"`
+
+	//BurstSize defines burst size in bytes.
+	BurstSize int `json:"burstSize"`
+
+	// CommittedBandwidth defines committed bandwidth in both directions specificd in Mb/s.
+	// Bandwidth is limited to line rate when the value configured is greater than line rate.
```

Comment:
Comment related to the comments left in terraform-provider-vcd PR to make sure it's correct.

Reply:
Thanks for catching it. 
This is the source of truth as these docs came from VCD docs - https://developer.vmware.com/apis/vmware-cloud-director/v37.1/data-structures/GatewayQoSProfile/
That is `Mb/s` for `CommittedBandwidth` and `bytes` for `BurstSize`